modules/monitoring-querying-metrics-for-all-projects-with-mon-dashboard.adoc

```diff
@@ -4,8 +4,11 @@
 // * virt/support/virt-prometheus-queries.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="querying-metrics-for-all-projects-as-an-administrator_{context}"]
-= Querying metrics for all projects as a cluster administrator
+[id="querying-metrics-for-all-projects-with-mon-dashboard_{context}"]
+// The following section will be included in the administrator section, hence there is no need to include "administrator" in the title
+= Querying metrics for all projects with the monitoring dashboard
+
+The {product-title} monitoring dashboard enables you to run Prometheus Query Language (PromQL) queries to examine metrics visualized on a plot. This functionality provides information about the state of a cluster and any user-defined workloads that you are monitoring.
```

Comment:
what does `monitoring dashboard` mean?
1. if it means dashboards under "Observe -> Dashboard", the metrics expression are pre-defined, we can not run PromQL queries, only can view the dashboards and the metrics expression from "Observe -> Metrics".
2. if it means "Observe -> Metrics", `monitoring dashboard` maybe not proper, how about `Metrics querying browser`?

Reply:
Sounds good! I am glad that these PRs are enabling us to find these existing issues in the docs :)

Reply:
@simonpasquier I think the change @juzhao points out  here makes a lot of sense, but I wanted to confirm with the engineering, WDYT? 

Also, I would suggest this title instead: `Querying metrics with the {OCP} web console` in addition the those changes.

Reply:
sounds good to me.